frontend/views/pages/Join.vue

```diff
@@ -1,79 +1,165 @@
 <template lang='pug'>
-//- test
-loading(theme='fullView' v-if='!ephemeral.contract.settings.groupName')
-page(pageTestName='dashboard' pageTestHeaderName='groupName' v-else='')
-  template(#title='') {{ L("You've been invited to join a group!") }}
-  .card
-    h1 {{ephemeral.contract.settings.groupName}}
-    p {{ephemeral.contract.settings.sharedValues}}
-
-    .c-join-grid-graphic(v-if='ephemeral.contract.members.length')
-      members-circle(:members='ephemeral.contract.members')
-        bars(:currency='ephemeral.contract.settings.mincomeCurrencySign' :history='ephemeral.contract.history' :mincome='+ephemeral.contract.settings.mincomeAmount')
-
-    p.error(v-if='ephemeral.errorMsg') {{ephemeral.errorMsg}}
-
-    .buttons
-      i18n.is-outlined(
-        tag='button'
-        data-test='declineLink'
-        v-on:click='decline'
-      )  No, thanks
-      i18n(
-        tag='button'
-        data-test='acceptLink'
-        v-on:click='accept'
-      ) Join Group
+div
+  img.c-logo(src='/assets/images/logo-transparent.png')
+
+  loading(theme='fullView' v-if='isStatus("LOADING")')
+  .c-page(v-else)
+    div(v-if='isStatus("SIGNING") || isStatus("LOGGING")')
+      .c-header
+        .c-avatars
+          avatar.c-avatars-group(
+            v-if='ephemeral.invitation.groupPicture'
+            :src='ephemeral.invitation.groupPicture'
+          )
+          avatar.c-avatars-creator(
+            v-if='ephemeral.invitation.creatorPicture'
+            :src='ephemeral.invitation.creatorPicture'
+          )
+        h1.title.is-1(data-test='groupName') {{ ephemeral.invitation.groupName }}
+        p.has-text-1(data-test='invitationMessage') {{ ephemeral.invitation.message }}
+      .card
+        form-signup(v-if='isStatus("SIGNING")' @submitSucceeded='accept')
+        form-login(v-else @submitSucceeded='accept')
+
+      p.c-switchEnter(v-if='isStatus("SIGNING")')
+        i18n Already have an account?
+        | &nbsp;
+        i18n.link(tag='button' @click='pageStatus = "LOGGING"' data-test='goToLogin') Log in
+      p.c-switchEnter(v-else)
+        i18n Not on Group Income yet?
+        | &nbsp;
+        i18n.link(tag='button' @click='pageStatus = "SIGNING"' data-test='goToSingup') Create an account
+
+    group-welcome.c-welcome(v-else-if='isStatus("WELCOME")')
+
+    .c-broken(v-else-if='isStatus("INVALID") || isStatus("EXPIRED")')
+      svg-broken-link.c-svg
+      i18n.title.is-1(
+        v-if='isStatus("EXPIRED")'
+        tag='h1'
+        :args='LTags()'
+      ) Oh no! {br_} Something went wrong.
```

Comment:
Typically there are no spaces before and after a `<br/>`, it's the difference between:

```
Oh no![ ]
[ ]Something went wrong.
```

And

```
Oh no!
Something went wrong.
```

Are we sure they need to be there? If not, on line 46 is the same issue

Reply:
Removed the second space, the first I can't join `!{br_}` because of the following error:
> 41:14  String interpolation operators must not be used